sass/base/components/just-in-time-block/_just-in-time.scss

```diff
@@ -7,10 +7,17 @@
 
 .form__row--just-in-time-block {
 
-  .toggle-link {
+  .link.toggle-link {
     position: relative;
     font-size: 18px;
-    transition: all 0.5s ease;
+    border-bottom: 1px solid #666;
+    font-family: $font-medium;
+    color: #666;
+
+    &:focus,
+    &:hover {
+      border-bottom: 1px solid #666;
```

Comment:
we'll need to put a `text-decoration: none;` in I think.. donate will have a double line otherwise

Reply:
Can do